content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-tool.md

```diff
@@ -154,4 +154,13 @@ python\python.cmd Gems\AWSCore\Code\Tools\ResourceMappingTool\resource_mapping_t
 
 ## Troubleshooting
 
-For help troubleshooting, check the resource mapping tool logs that are generated in `<ENGINE_ROOT>\Gems\AWSCore\Code\Tools\ResourceMappingTool\resource_mapping_tool.log`.
+### Check the logs
+Check the resource mapping tool logs that are generated in `<ENGINE_ROOT>\Gems\AWSCore\Code\Tools\ResourceMappingTool\resource_mapping_tool.log`.
+
+### Unable to call a resource
+Check the [resource mapping file](/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md) is configured correctly. Ensure the lookup name is defined as expected and it maps to the expected attributes. 
+
+Errors of the form ```Cannot determine region from the url``` in the logs indicate the wrong region has been set for the resource.
+* Double check that mapping entry's resource and account id are correct.
+* Double check that the resource type and name/id are as correct.
+* Ensure the mapping file has the correct global attributes set for region and accountid.
```

Comment:
Add another check like:
[Optional] Ensure the mapping file has the correct override attributes set for region and accountid if resource entry has.

Reply:
Already covered in that first line, but will update to " Double check that the resource's mapping entry has the correct region and account id, if overriding the global region and account attributes."